hc/accounts/tests/test_login.py

```diff
@@ -16,21 +16,26 @@ def test_it_sends_link(self):
         session["welcome_code"] = str(check.code)
         session.save()
 
-        form = {"email": "alice@example.org"}
-
-        r = self.client.post("/accounts/login/", form)
-        assert r.status_code == 302
+        form = {"email": "joan.awinja@andela.com"}
+        resp = self.client.post("/accounts/login/", form)
+        # self.assertEqual(resp.status_code, 302)
+        self.assertRedirects(resp, "/accounts/login_link_sent/")
 
+        
         ### Assert that a user was created
-
+        user = User.objects.get(email=form["email"])
+        self.assertTrue(user)
```

Comment:
Still not satisfied with the usage of `assertTrue` here, also in some tests below